### ISSUE TITLE
Error with sourcemaps

### ISSUE THREAD

**Issue by tomitrescak** [TypeStrong/ts-node#92]
Hi, I'm trying to run this with mcha but am getting following error:

```
/Users/tomi/Documents/Programming/Github/Mantra-Tran/tests/node_modules/ts-node/node_modules/source-map-support/source-map-support.js:407
      var hasStack = (arguments[1] && arguments[1].stack);
                                                  ^
SyntaxError:

npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "testonly"
npm ERR! node v5.9.0
npm ERR! npm  v3.7.3
npm ERR! code ELIFECYCLE
npm ERR! mantra-demo@0.3.1 testonly: `mocha --require ts-node/register .scripts/mocha_boot.js src/client/**/tests/**/*.ts`
```

I'm running mocha as following `mocha --require ts-node/register .scripts/mocha_boot.js src/client/**/tests/**/*.ts`

My sources and tsconfig are in the child directory "src".

**Comment by blakeembrey** [comment]
I'm not sure how much I can do with this. Have you considered logging what the error is? Seems odd that it just says `SyntaxError` and is pointing to a seemingly invalid location.

**Comment by arolson101** [comment]
I'm seeing this too.  It comes up when expect (from chai) is failing; that is, ```expect("foo").to.equal("bar")``` causes the error

**Comment by arolson101** [comment]
Oddly enough, if I add ```delete (global as any)["XMLHttpRequest"];``` after the ```jsdom()``` call then it works fine

**Comment by neekey** [comment]
@arolson101 your approach works for me, below is my setup file for jsdom:

```js
const jsdom = require('jsdom').jsdom;

global.document = jsdom('<!doctype html><html><body></body></html>');
global.window = document.defaultView;
global.navigator = global.window.navigator;

const propertyList = Object.keys(global.document.defaultView);
propertyList.forEach(function(property) {
  if (typeof global[property] === 'undefined' && property !== 'XMLHttpRequest') {
    global[property] = global.document.defaultView[property];
  }
});

global.documentRef = document;

```

**Comment by alloy** [comment]
This should be fixed by #97.

**Comment by blakeembrey** [comment]
Ok, I'll close it for now and assume I fixed it unless someone says otherwise. That fix was commit less than two weeks after this issue was opened though, so I guess the newer comments were on an older version still, but the original author never mentioned JSDom which others have.
